src/components/BorrowerProfile/LendCta.vue

```diff
@@ -484,6 +484,12 @@ export default {
 				this.cycleStatsSlot();
 			}
 		},
+		unreservedAmount(newValue, previousValue) {
```

Comment:
This is just for backwards compatibility a little later down the road. Generally, the drop-down will be hidden for loans with less than $25 remaining.